src/components/landing/LandingPageOneToOne.module.css

```
@@ -0,0 +1,823 @@
+  z-index: 98;
+  border: none;
+}
+
+.menuOverlay.active {
+  opacity: 1;
+  visibility: visible;
+}
+
+/* Intentionally empty: used as a toggle class with CSS-module selectors like `.mobileMenu.active` */
+.active {
+}
```

Stylelint (17.12.0)

[error] Empty block (block-no-empty)

(block-no-empty)